src/Tools/Altinn.Notifications.Tools.RetryDeadDeliveryReports/README.md

```
@@ -0,0 +1,73 @@
+   ```
+
+Or run from the solution root:
+   ```bash
+   dotnet run --project src/Tools/Altinn.Notifications.Tools.RetryDeadDeliveryReports/Altinn.Notifications.Tools.RetryDeadDeliveryReports.csproj
+   ```
+
+4. The tool will:
+- Retrieve dead delivery reports from the database within the specified ID range
+- Transform them into event grid messages
+- Post the messages to the configured webhook endpoint
+   - Log the processing results to the console
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)